MongoDatabase.py

```diff
@@ -41,14 +41,20 @@ def setCafeteriaMenu(self, allMealsInFile):
             assert isinstance(eachMeal, MealContainer)
             self.db['cafeteriaMenu'].update({'date':eachMeal.date}, eachMeal.toJson(), upsert=True)
 
-
-
+    # TODO refactor the next 3 functions!
     def insertCafeteriaRating(self, day, month, year, dinnerName, rating, remoteIp, datetime):
         self.db['cafeteriaRating'].insert({'day':day, 'month':month, 'year':year, 'dinnerName':dinnerName, 'rating':rating, 'remoteIp':remoteIp, 'lastUpdateTime': datetime})
 
     def insertCafeteriaRatingByMealId(self, mealId, rating, remoteIp, datetime):
         self.db['cafeteriaRating'].insert({'mealId':mealId, 'rating':rating, 'remoteIp':remoteIp, 'lastUpdateTime': datetime})
 
+    def insertCafeteriaRatingCommentByMealId(self, mealId, rating,

```

Comment:
why not insert an empty string instead? so we can use only one function when inserting to 'cafeteriaRating' collection.